gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```

Comment:
While this could be updated to 7.4.2 today, I'd rather just update to 7.5 when it becomes available (since using 7.3.x or higher gets us Java 17 compatibility, but we would need the unreleased Gradle 7.5 to support Java 18).